src/lib/AppRegistry.tsx

```diff
@@ -13,29 +14,30 @@ interface Props extends ViewProperties {
   trigger1pxScrollHack?: boolean
 }
 
+// Seeing as we're wrapping relay containers, we strip e.g. "Relay(Artist)" to be "Artist"
+// Probably move these methods to their own place
+// TODO: Add segment, add dev toggle between console and segement
+@track(props => ({ page: props.component.displayName.match("\\((.*?)\\)")[1] }), {
+  dispatch: data => console.log(data),
+})
 class RootContainer extends React.Component<Props, {}> {
   state: { retrying: boolean }
   component: Relay.RelayContainerClass<any>
   route: Relay.Route
   renderFetched?: Relay.RootContainerProps["renderFetched"]
-  forceFetch: boolean
```

Comment:
This should not be removed. This prop allows the Inbox to indicate that Relay should always fetch data from the server, even if it has it caches.